### PR TITLE
Task yc/j222 add library for popuper to handle screen overlapping

### PR DESCRIPTION
**Pull request by EugeniusGR**
image

### CODE REVIEW

File: frontend/src/components/common/header/header.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
-import { useAppSelector, useDispatch, useVisible, useState } from 'hooks/hooks';
+import { useAppSelector, useDispatch, useState } from 'hooks/hooks';
 import { AppRoute, ButtonType } from 'common/enums/enums';
 import { RootState } from 'common/types/types';
 import { Button, Link, ImageWrapper } from 'components/common/common';
 import { auth as authActions } from 'store/actions';
 import logo from 'assets/img/logo.svg';
 import headerBell from 'assets/img/header-bell.svg';
 import { getAllowedClasses } from 'helpers/helpers';
+import Popup from 'reactjs-popup';
```
Comment (what1s1ove) [inline frontend/src/components/common/header/header.tsx]: we can't import it directly
we need to wrap it in the `common` components folder

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by JuliaDzyuba — DISMISSED** [review]

**Review by what1s1ove — CHANGES_REQUESTED** [review]
Check the modal component, and do something like this

File: frontend/src/components/common/common.ts [hunk 2]
```diff
@@ -25,3 +25,4 @@ export { default as Player } from './player/player';
 export { default as ImageWrapper } from './image-wrapper/image-wrapper';
 export { default as Modal } from './modal/modal';
 export { default as ConfirmPopup } from './confirm-popup/confirm-popup';
+export { default as PopUp } from './popuper/popuper';
```
Comment (what1s1ove) [inline frontend/src/components/common/common.ts]:
```suggestion
export { default as Popuper } from './popuper/popuper';
```

File: frontend/src/components/common/header/popus.styles.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.popup-arrow {
+  color: var(--color-header-background);
+  filter: brightness(1.2);
+}
+
+.popup-content {
+  position: absolute;
+  top: 20px;
+  left: 20px;
+}
```
Comment (what1s1ove) [inline frontend/src/components/common/header/popus.styles.scss]:
```suggestion
```
move it to `frontend/src/components/common/header/styles.module.scss`
just use `global` selector

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**